inst/report/scripts/run_molevolvr_pipeline.R

```diff
@@ -0,0 +1,1296 @@
+    for (x in acc_files) {
+      accnum <- paste(readLines(x), collapse = ",")
+      system(sprintf(
+        "curl -X GET --header 'Accept:text/x-fasta' 'https://www.ebi.ac.uk/proteins/api/proteins?accession=%s' >> %s",
+        accnum, outfile
+      ))
+
+    }
+  }
+
+  # Remove temporary files
+  cat("\nRemoving temporary files\n")
+  system(sprintf("rm %s/acc*", outdir))
+
+  # Print end message
+  cat("#####################\n")
+  cat("END OF EDIRECT SEARCH\n")
+  cat("#####################\n")
+}
+
+
+cleanup_blast <- function(infile_blast, acc2info, prefix, wblast = F) {
+
+  outdir <- dirname(infile_blast)
+  # Load and clean acc2info file
+  acc2info_out <- fread(input = acc2info, sep = "\t",
+                        header = TRUE, fill = TRUE) %>%
+    mutate(FullAccNum = gsub("\\|", "", FullAccNum)) %>%
+    mutate(FullAccNum = gsub(".*[a-z]", "", FullAccNum))
+
+  # If using web-blast output
+  if (wblast == TRUE) {
+    blast_out <- fread(input = infile_blast, sep = "\t", header = FALSE,
+                       col.names = web_blastp_hit_colnames, fill = TRUE)
+    cleanedup_blast <- blast_out %>%
+      mutate(AccNum = gsub("\\|", "", AccNum)) %>%
+      mutate(AccNum = gsub(".*[a-z]", "", AccNum)) %>%
+      mutate(PcIdentity = round(as.double(PcIdentity), 2))
+
+    # Merge cleaned blast output with acc2info
+    cleanedup_blast <- merge(cleanedup_blast, acc2info_out,
+                             by.x = "AccNum",
+                             by.y = "FullAccNum",
+                             all.x = TRUE)
+    names(cleanedup_blast)[names(cleanedup_blast) == "Species.y"] <- "Species"
+
+    # Additional calculations for PcPositive
+    cleanedup_blast <- cleanedup_blast %>%
+      mutate(PcPosOrig = as.numeric(PcPosOrig)) %>%
+      mutate(AlnLength = as.numeric(AlnLength)) %>%
+      mutate(PcPositive = PcPosOrig)
+
+  } else if (wblast == FALSE) {
+    # If using classic-blast output
+    blast_out <- read_tsv(file = infile_blast, col_names = cl_blast_colnames)
+    cleanedup_blast <- blast_out %>%
+      mutate(AccNum = gsub("\\|", "", AccNum)) %>%
+      mutate(AccNum = gsub(".*[a-z]", "", AccNum)) %>%
+      mutate(Species = gsub(";.*$", "", Species)) %>%
+      mutate(PcIdentity = round(PcIdentity, 2)) %>%
+      mutate(PcPositive = round((PcPosOrig * AlnLength / QLength), digits = 2))
+
+    # Merge cleaned blast output with acc2info
+    cleanedup_blast <- merge(cleanedup_blast, acc2info_out,
+                             by.x = "AccNum",
+                             by.y = "FullAccNum",
+                             all.x = TRUE) %>%
+      select(-Species.x, -TaxID.x)
+    names(cleanedup_blast)[names(cleanedup_blast) == "Species.y"] <- "Species"
+  }
+
+  # TaxID to lineage mapping
+  cleanedup_blast$TaxID <- as.integer(cleanedup_blast$TaxID)
+  lineage_map <- fread("~/awasyn/new_trial/lineage_lookup.txt",
+                       header = TRUE, fill = TRUE,
+                       colClasses = lineage_map_cols)
```

Comment:
Similar to my comment in `ipr2viz.R`, I'd change this to something like the following after moving `lineage_lookup` into `inst/common_data`:

```suggestion
  lineage_map <- fread(
    system.file("common_data", "lineage_lookup.txt", package = "MolEvolvR", mustWork = TRUE),
    header = TRUE, fill = TRUE, colClasses = lineage_map_cols)
```